svc-bgs-api/src/Gemfile

```diff
@@ -0,0 +1,74 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.6"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.0.4", ">= 7.0.4.3"
```

Comment:
The dependencies in this file can probably be removed to address SecRel alerts. Check with Yang.